.github/workflows/build-project.yaml

```diff
@@ -28,25 +29,30 @@ jobs:
 
           case "${GITHUB_EVENT_NAME}" in
             pull_request)
-              config_data=('codesign:false' 'notarize:false' 'package:false' 'config:RelWithDebInfo')
-              if gh pr view ${{ github.event.number }} --json labels \
+              config_data=('codesign:false' 'notarize:false' 'package:false' 'config:RelWithDebInfo' 'flatpakArch:["x86_64"]')
```

Comment:
I'd rather have this step create a bespoke matrix variable instead of putting this in `config_data`, as described in the GitHub action documentation:

https://docs.github.com/en/enterprise-cloud@latest/actions/learn-github-actions/expressions#example-returning-a-json-object

I'm fine with `flatpakArch` being set up as a bespoke output variable depending on GitHub event.

Reply:
Assigning a whole matrix looks unnecessary, making the JSON object (only an array) more complex (an array in an object) where there is only one variable does not look worth it.

I was aware of this feature to set up a matrix but for one variable, it looks too much.